src/Microsoft.ML.Dnn/DnnUtils.cs

```diff
@@ -494,5 +498,11 @@ public Tensor[] Run()
 
         }
 
+        internal static string GetTemporaryDirectory()
```

Comment:
>internal [](start = 8, length = 8)

private

Reply:
Cannot change to private as it is used in ImageClassificationTrainer.cs, and thus must remain internal

---
In reply to: [340255936](https://github.com/dotnet/machinelearning/pull/4410#discussion_r340255936) [](ancestors = 340255936)